packages/terra-avatar/tests/jest/Avatar.test.jsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import Avatar from '../../src/Avatar';
+import exampleProfilePhoto from '../assets/150x150.jpg';
+
+const propsWarningMsg = 'Only one of the props: [image, initials] should be supplied.';
+
+describe('Avatar', () => {
+  // Snapshot Tests
+  it('should render user avatar when variant is user', () => {
+    const avatar = <Avatar variant="user" ariaLabel="user" />;
+    const wrapper = render(avatar);
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  it('should render facility avatar when variant is facility', () => {
+    const avatar = <Avatar variant="facility" ariaLabel="facility" />;
+    const wrapper = render(avatar);
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  it('should render two intials avatar when two initials are passed in', () => {
```

Comment:
*initials

Reply:
Updated in this [commit](https://github.com/cerner/terra-core/commit/901237e7a6938c3655ffa65a772ace2bb6d7144e).